arches/install/ubuntu_setup.sh

```diff
@@ -40,9 +41,14 @@ function install_postgres {
 }
 
 function install_yarn {
-  wget --quiet -O - https://deb.nodesource.com/setup_16.x | sudo -E bash -
   sudo apt-get update
-  sudo apt-get install -y nodejs
+  sudo apt-get install -y ca-certificates curl gnupg
+  sudo mkdir -p /etc/apt/keyrings
+  curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
```

Comment:
We are calling curl here, but it is installed on line 52. Maybe line 52 should be moved before this line - just to be safe, or just remove line 52 altogether.